### DISCUSSION TITLE
Difference in importing Windows Event Logs via Wazuh vs Winlogbeat & OSquery

### DISCUSSION THREAD

**Discussion by mciantar** [Security-Onion-Solutions/securityonion#3742] [Unsupported Versions] [answered]
1. What is the difference in how windows eventlogs are processed when these are injested via Wazuh Agent instead of Winlogbeat or OSQuery? This goes in addition to additional logs, like Powershell Script Logging as well as Sysmon.

2. What is the meaning of the follwing in the documentation (https://docs.securityonion.net/en/2.3/beats.html), in relation to Winlogbeat?
If you are shipping Sysmon logs, confirm that your Winlogbeat configuration does NOT use the Elastic Sysmon module as Security Onion will do all the necessary parsing.
Can someone explain this further please in reference to how the config should/shouldn't look for winlogbeat to be sure I am doing this properly?

**Comment by defensivedepth** [comment, marked as answer]
There should be very little differences. All three shippers (Wazuh, Winlogbeat, Osquery) normalize the logs and then pass them to the same Windows Eventlog parser. If you see something odd, please let us know so that we can get it fixed.

RE: #2. Best thing to do is just download Winlogbeat from the Downloads page in your Security Onion Console. That is the one you need and it will not use the Elastic Sysmon module.
